src/main/java/com/github/danielflower/mavenplugins/release/LocalGitRepo.java

```diff
@@ -20,13 +20,88 @@
 public class LocalGitRepo {
 
     public final Git git;
-    private final String remoteUrl;
```

Comment:
This field is now embedded in the strategy objects by the builder, so doesn't need to be in this object.